python/mxnet/image/image.py

```diff
@@ -133,6 +135,9 @@ def imdecode(buf, *args, **kwargs):
     <NDArray 224x224x3 @cpu(0)>
     """
     if not isinstance(buf, nd.NDArray):
+        if sys.version_info[0] == 3 and not isinstance(buf, (bytes, np.ndarray)):
+            raise ValueError('buf must bytes or numpy.ndarray,'
+                             'if you input str,please convert it to bytes')
```

Comment:
nit: suggest rephrase: "if you would like to input type str, please convert to bytes"